hera_mc/cm_hookup.py

```diff
@@ -121,35 +121,60 @@ def add_correlator_levels(self, pol):
         self.columns[pol].append('level')
         self.level[pol] = 'N/A'
 
-    def get_part_info(self, part_name):
+    def get_part_in_hookup_from_type(self, part_type, include_revs=False, include_ports=False):
         """
         Retrieve the value for a part name from a hookup
```

Comment:
Should this docstring refer to "a part type" rather than "a part name"? It seems that you're distinguishing between types and names, but that doesn't seem to be consistent throughout...